websites/B/BloomHost/presence.ts

```diff
@@ -0,0 +1,91 @@
+const presence = new Presence({
+		clientId: "873728851052752896"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	{ hostname, pathname } = document.location;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "bloom_logo",
+		startTimestamp: browsingTimestamp
+	};
+
+	switch (hostname) {
+		case "bloom.host":
+			presenceData.smallImageKey = "globe-solid";
+			presenceData.details = "Viewing the main website";
+			if (pathname.includes("minecraft"))
+				presenceData.state = "Reading: Minecraft Plans";
+			else if (pathname.includes("vps"))
+				presenceData.state = "Reading: VPS Plans";
+			else if (pathname.includes("duck-panel"))
+				presenceData.state = "Reading: About the Duck Panel";
+			else if (pathname.includes("about-bloom"))
+				presenceData.state = "Reading: About Bloom";
+			break;
+		case "docs.bloom.host":
+			presenceData.details = "Viewing the docs";
+			presenceData.state = `Reading: ${
+				document.querySelector("div.text--center > header > h1").textContent
+			}`;
+			break;
+		case "mc.bloom.host" && "demo.bloom.host":
```

Comment:
this is not valid JS

Reply:
how?

Reply:
You’d need to make another case in the line right above this to account for both strings. Currently this is only checking for the last one and you should’ve realized this when testing